store/docs/soap.txt

```diff
@@ -4732,12 +4732,13 @@ Api to get list of available contact backup files
 API to search for mail items and take some bulk action on the search result
 <SearchActionRequest>
 </SearchRequest>
-<BulkAction op={action-to-be-taken-on-search-result} l={folder-path}/>
+<BulkAction op={action-to-be-taken-on-search-result} l={folder-path} read={true/false}/>
```

Comment:
There is some change in request now. Removed the 'read' attribute altogether. Added two new op values: 'read' and 'unread'
This change is made to avoid the ambiguity with 'flag' action